styles/style.css

```
@@ -7,4 +7,115 @@ light blue: #1c5380
 body {
   font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
   margin: 0;
+  background-image: url("../images/bgtile.png");
+  background-repeat-y: no-repeat;
 }
+
+.general-container {
+  margin: 10%;
+  margin-top: 5%;
+  height: 500px;
+}
+
+.left-bar {
+  height: 500px;
+  display: block;
+  float: left;
+  position: fixed;
+}
+
+.right-bar {
+  height: 500px;
+  width: 70%;
+  float: right;
+  display: block;
+  position: relative;
+
+}
+
+footer {
+  display: block;
+  text-align: center;
+}
+
+li {
+  display: inline-block;
+  margin-left: 25px;
+}
+
+a {
+  text-decoration: none;
+}
+
+footer a {
+  color: #3f729b;
+  font-weight: bold;
+}
+
+.copyR {
+  color: grey;
+  font-weight: bold;
+}
+
+.leftBox {
+  margin: 5%;
+}
+
+.login div {
+  display: inline-block;
+  text-align: center;
+}
+
+.login {
+  text-align: center;
+}
+
+.loginButton {
+  padding: 10px;
+  margin-right: 30px;
+  border: 2px solid #325877;
+  border-radius: 5px;
+  float: right;
+  background: linear-gradient(#78afdd, #1c5380);
+}
+
+.loginButton a {
+  color: white;
+  border-radius: 5px;
+  font-weight: 600;
+  text-decoration: none;
+  font-weight: 600;
+  padding: 5px 10px;
+}
+
+
+.white-box {
+  background-image: url('../images/frame.png');
+  background-repeat: no-repeat;
+  background-size: 100%;
+  padding: 40px;
+  border-radius: 5px;
+}
+
+.white-box h2 {
+  color: #325877;
+  margin: 0;
+  line-height: 1.3em;
+  padding-bottom: 10px;
+  font-size: 26px
+}
+
+.white-box p {
+  font-size: 17px;
+  text-align: justify;
+}
+
+.downloadButtons{
+  text-align: center;
+
+}
+
+.downloadButtons img{
+  margin-right:10px ;
+  margin-left: 10px;
+}
```

No encontré la clase relacionada a la imagen del boton de login, pero una pequeña sugerencia:
```suggestion
}

.button-image{
  vertical-align: middle; /* o top;*/
}
```

Lo que hace vertical align es alinear la imagen respecto a la caja donde está contenido.

No lo vemos en clase pero es un tip extra.